LEGO1/mxdirectdraw.h

```diff
@@ -5,138 +5,137 @@
 #include <ddraw.h>
 #include <Windows.h>
 
-BOOL g_is_PALETTEINDEXED8 = 0;
-BOOL g_isInsideError = FALSE;
+extern BOOL g_is_PALETTEINDEXED8;
 
 //size 0x880
 class MxDirectDraw
 {
 public:
-    typedef void (*ErrorHandler)(const char*, HRESULT, void*);
+  typedef void (*ErrorHandler)(const char*, HRESULT, void*);
```

Comment:
Might want to be explicit about calling convention here:
`typedef void (__cdecl *ErrorHandler)(const char*, HRESULT, void*);`

Reply:
I don't think it's necessary. As long as it not a class functions it must have C calling convention.(unless compiler flags says other)

Reply:
> (unless compiler flags says other)

You correctly identified the case I tried to protect against; `/Gr` or `/Gz`. It would make for... interesting debugging. How loud do you think you would curse if you were in such a project, and finally realized why the stack was smashed? :-)

Anyway, no biggie. Those flags are few and far between. I just wanted to flag for potential blood pressure increases down the line.